src/ralph/cross_validator/__main__.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+
+
+def main():
+    if 'cross_validator' not in os.environ.get('DJANGO_SETTINGS_MODULE', ''):
```

Comment:
yes - it is correct (but not perfect). the idea here is to enable local settings easily (ex. ralph.cross_validator.settings_local).